### PR TITLE
chore: theme improvements and no handouts handling

### PR DESCRIPTION
**Pull request by saeedbashir**
This PR enhances theme capability by using custom color pellets, giving more breathing space to logo, logo aspect fit, and handles when there aren't any handouts for a course.

### CODE REVIEW

**Comment by volodymyr-chekyrta** [comment]
@saeedbashir, Could you please add some screenshots for design review?

File: Course/Course/Presentation/Handouts/HandoutsUpdatesDetailView.swift [hunk 1]
```diff
@@ -27,12 +27,16 @@ public struct HandoutsUpdatesDetailView: View {
         router: CourseRouter,
         cssInjector: CSSInjector
     ) {
-        if handouts != nil {
+        let handoutsView = handouts != nil && announcements == nil
+        let noHandouts = handouts == nil && announcements == nil
+        
+        if handoutsView || noHandouts {
```
Comment (rnr) [inline Course/Course/Presentation/Handouts/HandoutsUpdatesDetailView.swift:33]: it appears than this condition can be replaced just with `announcements == nil`

**Review by rnr — APPROVED** [review]

File: Course/Course/Presentation/Handouts/HandoutsUpdatesDetailView.swift (hunk 1 above)
Comment (saeedbashir) [reply Course/Course/Presentation/Handouts/HandoutsUpdatesDetailView.swift]: Done

**Review by rnr — APPROVED** [review]

**Comment by saeedbashir** [comment]
> @saeedbashir, Could you please add some screenshots for design review?

@volodymyr-chekyrta There won't be any design changes for openedX, the changes will only reflect with different color values, for openedX the values are the same.

**Review by volodymyr-chekyrta — APPROVED** [review]

**Pull request merged**